covigator/database/model.py

```diff
@@ -1150,3 +1152,43 @@ class Lineages(Base):
     variant_under_investigation = Column(Boolean, default=False)
     parent_lineage_id = Column(String)
     tags = Column(String)
+
+    variants = relationship(
+        "LineageDefiningVariants",
+        secondary=LINEAGE_SITES_JUNCTION_TABLE_NAME,
+        back_populates='constellations',
+    )
+
+
+class LineageDefiningVariants(Base):
+    """
+    Store lineage defining mutations
+    """
+    __tablename__ = CONSTELLATION_SITES_TABLE_NAME
+
+    variant_id = Column(String, primary_key=True)
+    variant_type = Column(String)
+    protein = Column(String)
+    position = Column(Integer, index=True)
+    reference = Column(String)
+    alternate = Column(String)
+    ambiguous_alternate = Column(Boolean, default=False)
+    annotation = Column(String, default="missense")
+    hgvs = Column(String)
+    variant_level = Column(String)
```

Comment:
I would make this an enum to make the model more robust.

Reply:
Which column would you make an enum?

Reply:
`variant_level`, the rationale is that this column can take only two values. If you keep it as a string then you open the door to typing errors in the python code or in any SQL data manipulation. If you use an enum you will have the constants at hand in Python and the DB will make a further check on data validity.